### ISSUE TITLE
TupleTarget

### ISSUE THREAD

**Issue by gabrielgonzaga** [datails/ruigi#1]
Define a way to create tuple targets. E.g.

Say my task creates an encoder, and I want to return the encoding object and the transformed data frame:

@inherit_list(ETLTask)
class EncodeCat(Task):
    target_type = PickleTarget, ParquetTarget

    def easy_run(self, inputs):
        df = inputs[0]
        df, encs = encode_func(df)
        return df, encs

One would be a Pickle and the other a Parquet

**Comment by rafarui** [comment]
This is against Luigi idea. Basically it is "one task, one output"

we could handle that on that "complete"method to guarantee that all targets are there. maye it is a good feature.

**Comment by gabrielgonzaga** [comment]
Yes, I was thinking st. like that, make sure this kind of target deals correctly with all .complete Targets inside it
